wagtail/core/models.py

```diff
@@ -1073,6 +1073,13 @@ def get_verbose_name(cls):
         # except this doesn't convert any characters to lowercase
         return capfirst(cls._meta.verbose_name)
 
+    @classmethod
+    def get_page_description(cls):
+        """
+        Returns a page description if it's not set. eg "A multi-purpose web page".
```

Comment:
Typo?

```suggestion
        Returns a page description if it's set. eg "A multi-purpose web page".
```

Reply:
Bonus point for documentation – most content guidelines focused on readability suggest avoiding "eg" or other latin abbreviation and going for more plain language like "For example" instead.